utils/stacktrace/frame.js

```diff
@@ -24,11 +24,28 @@ class Frame {
    */
   constructor(name, source, line, column) {
     this.name = name;
-    this.source = source;
+    this.source = this.dedupeSource(source);
```

Comment:
Nit: should be a static method or just a module function.

Reply:
Doen.